File: lista_Logica_v2/emC/ex6.c
#include <stdio.h>

void main() {
    int num, i;
    
    printf("Digite um numero: ");
    scanf("%d", &num);
    
    printf("===== TABUADA DO %d =====\n", num);
    
    for (i = 1; i <= 10; i++) {
        printf("%d x %d = %d\n", num, i, num*i);
    }
}

File: lista_Logica_v2/ex6.py
# 6) Faça um programa que mostre a tabuada de qualquer número escolhido pelo usuário (considerar tabuada do número 1 ao 10).

num = int(input("Digite um número"))
print(f"===== TABUADO DO {num} =====")
for i in range(1, 11):
    print(f"{num} x {i} = {(num*i)}")
